### DISCUSSION TITLE
Help to change display name of mdx files in goldendict

### DISCUSSION THREAD

**Discussion by KamlaK10** [xiaoyifang/goldendict-ng#457] [Q&A]
Can somebody pls change the name of these dictionaries to English or German? I don't understand Chinese. It becomes very difficult like that to search for the dictionaries

These files are in .mdx format and I don't know  anything about compiling or modifying these files.  
I am just a regular language  learner. 
I found them on freemdict site.
I will provide the links to these files if somebody could help me out. Thanks a lot in advance.
![pons ger-eng](https://user-images.githubusercontent.com/89730239/230727037-fc27e84b-e871-40db-a562-51a32630ff97.png)
![business and eco german eng](https://user-images.githubusercontent.com/89730239/230727039-b2644d45-f644-4384-a461-f0392eb2f7de.png)
![economic and trade eng- ger](https://user-images.githubusercontent.com/89730239/230727040-bd3c9a06-22dc-4fcc-bded-4c411a947802.png)
![pons eng-ger](https://user-images.githubusercontent.com/89730239/230727041-b23632d6-aa82-4944-929d-5acf092c1387.png)

**Comment by xiaoyifang** [comment]
Dictionary making is not related to this repository

**Reply by KamlaK10** [reply]
Ok I am sorry where should I pose this question then?
Could you please guide me, where to direct this question?

 I found the right place I guess. I need to ask on issue section.  https://github.com/goldendict/goldendict/issues
Really sorry for wasting your time, sir/madam

**Comment by xiaoyifang** [comment]
https://xiaoyifang.github.io/goldendict-ng/custom_dictionary/#:~:text=zip%0A%20%20%20%20%20%E2%94%94%E2%94%80%E2%94%80%20Collins.dsl-,Override%20display%20name,-Some%20dictionary%20formats
